cortex/kernel_features/llm_device.py

```
@@ -0,0 +1,142 @@
+class Session:
+    id: str
+    model: str
+    messages: List[Dict] = field(default_factory=list)
+    prompt: str = ""
+    response: str = ""
+    temp: float = 0.7
+    max_tokens: int = 4096
+
+
+class MockLLM:
+    def complete(self, model, messages, max_tokens, temp, system=None):
```

Ruff (0.14.7)

Unused method argument: `model`

(ARG002)